10up/classifai
# Bump various dependencies

### Description of the Change

This PR will be a slow process of iterative dependency updates to see what all can be bumped across the stack and still have actions/tests pass.

### How to test the Change

Load PR locally and test ClassifAI activation, setup flows, language processing, image processing, and recommended content tasks still function as expected.

### Changelog Entry

> Security - Bumped various dependencies.

Note we can be more specific on the above once we determine when the various iterative commits here are stable enough to merge and then see what dependencies ultimately were updated.

### Credits

Props @jeffpaul.

### Checklist:
